azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -439,10 +451,24 @@ def get_module(self, device_id, module_id):
         """
         return self.protocol.service.get_module(device_id, module_id)
 
+    def get_modules(self, device_id):
```

Comment:
`get_all_modules`?

Reply:
I am using the convention of our APIs here.

---
In reply to: [349798734](https://github.com/Azure/azure-iot-sdk-python/pull/397#discussion_r349798734) [](ancestors = 349798734)